oremda/pipeline/runner/.env

```diff
@@ -0,0 +1,5 @@
+OREMDA_VAR_DIR=/home/cjh/run/oremda
+OREMDA_DATA_DIR=/home/cjh/data/oremda
+OREMDA_DIR=/home/cjh/work/source/oremda
+
```

Comment:
Should we put the `PLASMA_SOCKET` in here too, as:
```bash
PLASMA_SOCKET=${OREMDA_VAR_DIR}/plasma.sock
```

? I see you can refer to earlier variables like this.

Reply:
We can do for sure, but as we are adding the `.env` to the `.gitignore` it will be be up to each dev.

Reply:
Done, I will also remove the file